.github/workflows/cache-dependencies.yml

```diff
@@ -76,16 +76,49 @@ jobs:
         with: 
           repository: openmc-dev/openmc
           path: openmc
-
-      - name: Build OpenMC
+          
+      - name: Build OpenMC from scratch
```

Comment:
```suggestion
      - name: Build OpenMC from source
```

Reply:
Is this what you mean by "scratch" here?

Reply:
Yes!